docs/guide/security_checklist.md

```diff
@@ -4,65 +4,192 @@ description: Ensure that your Ibexa DXP installation is secure by following our
+- Is access to user data properly restricted, in accordance with GDPR?
+- Is access to Form Builder uploads managed properly? Files uploaded with the Form Builder are accessible to any user by default. If this doesn't suit you, restrict access to the Form Uploads folder.
+
+### Don't use "hide" for read access restriction
+
+The [visibility switcher](https://doc.ibexa.co/en/latest/content_management/locations/#location-visibility) is a convenient feature for withdrawing content from the frontend.
+It acts as a filter in the frontend by default.
+You can choose to respect it or ignore it in your code.
+It isn't permission-based, and doesn't restrict read access to content.
+Hidden content can be read through other means, like the REST API or GraphQL.
+
+If you need to restrict read access to a given content item, you could create a role that grants read access for a given [**Section**](https://doc.ibexa.co/en/latest/administration/content_organization/sections/) or [**Object State**](https://doc.ibexa.co/en/latest/administration/content_organization/object_states/), and set a different section or object State for the given content.
+Or use other permission-based [**Limitations**](https://doc.ibexa.co/en/latest/permissions/limitations/).
+
+### Minimize exposure
+
+Security should be a multi-layered exercise.
+It's wise to minimize what features you make available to the world, even if there are no known or suspected vulnerabilities in those features, and even if your content is properly protected by roles and policies.
+Reduce your attack surface by exposing only what you must.
+
+- If possible, make the back office unavailable on the open internet.
+- [Symfony FOSJsRoutingBundle](https://github.com/FriendsOfSymfony/FOSJsRoutingBundle) is required in those releases where it's included, to expose routes to JavaScript. It exposes only the required routes, nothing more. It's only required in the back office SiteAccess though, so you can consider blocking it in other SiteAccesses. You should also go through your own custom routes, and decide for each if you need to expose them or not. See the documentation on [YAML route definitions for exposure](https://github.com/FriendsOfSymfony/FOSJsRoutingBundle/blob/master/Resources/doc/usage.rst#generating-uris).
+- By default, a [Powered-By header](https://doc.ibexa.co/en/latest/update_and_migration/from_1.x_2.x/update_db_to_2.5/#powered-by-header) is set. It specifies what version of the DXP is running. For example, `x-powered-by: [[= product_name_exp =]] v4`. This doesn't expose anything that couldn't be detected through other means. But if you wish to obscure this, you can either omit the version number, or disable the header entirely by setting `enabled: false`.
+
+    ```yaml
+    ibexa_system_info:
+      system_info:
+        powered_by:
+          # major => v4 || minor => v4.6 || none
+          release: major
+          # true || false
+          enabled: false
```

Comment:
3.3 might use https://github.com/ezsystems/ez-support-tools/blob/2.3/src/bundle/DependencyInjection/Configuration.php so config name has changed.
I'm investigating closer…

Reply:
According to my comparison of https://github.com/ibexa/system-info/blob/v4.6.14/src/bundle/DependencyInjection/Configuration.php and https://github.com/ezsystems/ez-support-tools/blob/v2.3.12/src/bundle/DependencyInjection/Configuration.php only the extension name change.

```suggestion
    ezplatform_support_tools:
      system_info:
        powered_by:
          # major => v4 || minor => v4.6 || none
          release: major
          # true || false
          enabled: false
```